src/Routers/SettingsRouter.js

```diff
@@ -1,26 +1,30 @@
 import PromiseRouter from '../PromiseRouter';
 import * as middleware from "../middlewares";
 import { logger, configureLogger } from '../logger';
+import SettingsManager from '../SettingsManager';
 import winston from 'winston';
 
 export class SettingsRouter extends PromiseRouter {
   mountRoutes() {
     this.route('GET', '/settings', middleware.promiseEnforceMasterKeyAccess, (req) => {
       return Promise.resolve({
-        response: {
-          logLevel: winston.level
-        }
-      })
+        response: SettingsManager(req.config.applicationId).getUnlocked()
```

Comment:
I'd expect that the primary use of this endpoint would be to display the current settings in the dashboard, so I'd suggest returning all settings here.